js/index.js

```diff
@@ -450,6 +450,8 @@ function searchFeatureColumn() {
     }
     const featureFilter = function(data) {
       if(Array.isArray(data)) {
+        data = data.map(i => "(" + i);
+        data = data.map(i => i + ")");
```

Comment:
@siva19susi can this be substituted with

```js
data = data.map(i => `(${i})`);
```

Reply:
switched to this method in [b6395c2](https://github.com/srl-labs/yang-browser/pull/39/commits/b6395c231c78b81c4214aff2e0bc293d28a19183) along with formatting